src/Katanox/Model/Price.php

```diff
@@ -4,8 +4,8 @@
 
 class Price
 {
-    private float $amount;
-    private string $currency;
+    public float $amount;
```

Comment:
why are these public?
I believe making them private with getters/setters would be more appropriate, since it will be more consistent with the database

Reply:
Updated